### ISSUE TITLE
issue: MALWARE ATTEMPT message

### ISSUE THREAD

**Issue by benjaminmodayil** [react-hook-form/react-hook-form#11933]
### Version Number

7.37.0/6.15.8

### Codesandbox/Expo snack

N/A

### Steps to reproduce

I'm getting this error from the Netgear Nighthawk app:
![IMG_8738C49BC3DC-1](https://github.com/react-hook-form/react-hook-form/assets/12306850/6c5bea30-9db8-45ec-ad5b-31d59199b361)

Took me forever to figure out what BeekAI was and what connection it had to our codebase until I looked into the docs for `react-hook-form` and saw a link for it there.

This is VERY troubling that my router thinks there is a malware attempt from this library. Please look into this/let us know why this is happening.

Note: We are currently using two versions of react-hook-form as noted in this form as we've been moving from v6 to v7.

### Expected behaviour

No malware.

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
not related to this library.

**Comment by benjaminmodayil** [comment]
@bluebill1049 You're right, I've been able to reproduce this issue somewhat reliably. It's not the library itself, but it's the react-hook-form documentation site that triggers this error. I ran the site through virustotal.com and it comes out clear, but the linked site for BeekAI on the documentation for react-hook-form does not come out as clear https://www.virustotal.com/gui/domain/beekai.com. Maybe if what is being flagged on BeekAI's site is fixed, this error would go away for people that visit the documentation for react-hook-form.
